src/System.ServiceModel.Http/src/System.ServiceModel.Http.builds

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <ItemGroup>
+    <Project Include="System.ServiceModel.Http.csproj" />
+    <Project Include="System.ServiceModel.Http.csproj" >
+      <TargetGroup>net46</TargetGroup>
+    </Project>
+    <Project Include="System.ServiceModel.Http.csproj" >
```

Comment:
I don't think that is necessary. In general I think the .builds file for a given library should remain agnostic to its dependencies and only build for the configurations that it needs itself. Given these are a facade I would expect you only need the "Empty", "net46", and "netcore50" for this guy. It does mean however you should explicitly pick a version of S.P.SM to reference. See https://github.com/dotnet/corefx/blob/master/src/System.Collections/src/System.Collections.csproj#L20 where I explicitly always pick the Windows_NT version of its dependency. It really doesn't matter which you pick as long as the public surface is the same which it should be in this case.